tests/integration/golang/mlflow/namespace/flows/metric_test.go

```diff
@@ -0,0 +1,441 @@
+//go:build integration
+
+package flows
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type MetricFlowTestSuite struct {
+	suite.Suite
+	helpers.BaseTestSuite
+}
+
+// TestMetricFlowTestSuite tests the full `metric` flow connected to namespace functionality.
+// Flow contains next endpoints:
+// - `GET /metrics/get-history`
+// - `GET /metrics/get-history-bulk`
+// - `POST /metrics/get-histories` - TODO:dsuhinin we need firstly to create proper decoder.
+func TestMetricFlowTestSuite(t *testing.T) {
+	suite.Run(t, new(MetricFlowTestSuite))
+}
+
+func (s *MetricFlowTestSuite) SetupTest() {
+	s.BaseTestSuite.SetupTest(s.T())
+}
+
+func (s *MetricFlowTestSuite) TearDownTest() {
+	assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+}
+
+func (s *MetricFlowTestSuite) Test_Ok() {
+	tests := []struct {
+		name           string
+		setup          func() (*models.Namespace, *models.Namespace)
+		namespace1Code string
+		namespace2Code string
+	}{
+		{
+			name: "TestInScopeOfTwoCustomNamespaces",
```

Comment:
Again, I'd reduce these names down to the bare essence: "TwoCustomNamespaces", "OneDefaultOneCustom", etc